packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/EditInvoiceDetails.tsx

```diff
@@ -229,28 +229,23 @@ const EditInvoiceDetailsContent = ({
               });
             })}
           >
-            <Stack spacing={2} sx={{ mt: 2 }}>
-              <Typography variant="h2" sx={{ mb: 2 }}>
-                {t(i18n)`Edit invoice ${invoice.id}`}
-              </Typography>
-              <Stack direction="column" spacing={4}>
-                <CustomerSection disabled={isLoading} />
-                <EntitySection
-                  disabled={isLoading}
-                  hidden={['purchase_order']}
-                />
-                <ItemsSection
-                  actualCurrency={actualCurrency}
-                  onCurrencyChanged={setActualCurrency}
-                />
-                <PaymentSection disabled={isLoading} />
-                <ReminderSection
-                  disabled={isLoading}
-                  onUpdateOverdueReminder={onEditOverdueReminder}
-                  onUpdatePaymentReminder={onEditPaymentReminder}
-                  onCreateReminder={onCreateReminder}
-                />
-              </Stack>
+            <Typography variant="h1" sx={{ mb: 7 }}>
```

Comment:
Same here, heading should not care about if there is a free space required or not.

Reply:
@radist2s Changed the code as requested.